Cargo.toml

```diff
@@ -32,7 +32,7 @@ bytelines = "2.4"
 byteorder = "1.4"
 caseless = "0.2"
 chardetng = "0.1"
-chrono = "0.4"
+chrono = { version = "0.4", features = ["serde"] }
```

Comment:
no action right now, but we should probably switch to `time` instead of `chrono`. As I understand `chrono` isn't being updated and depends on an old version of `time` anyway